scripts/create_changelog.sh

```diff
@@ -15,42 +15,37 @@ GITHUB_URL=https://api.github.com/repos/${REPOSITORY}
 GITHUB_AUTH_HEADER="Authorization: token ${GITHUB_TOKEN}"
 CHANGELOG_FILE="CHANGELOG.md"
 
-if [ "${PREVIOUS_RELEASE}"  == "" ]
-then
-  PREVIOUS_RELEASE=$(git describe --tags --abbrev=0)
+if [ "${PREVIOUS_RELEASE}" == "" ]; then
```

Comment:
(nit) Can you add more comments in this file to explain whats happening at each step?